c10/core/DispatchKey.cpp

```diff
@@ -40,6 +40,8 @@ const char* toString(DispatchKey t) {
       return "ComplexCUDATensorId";
     case DispatchKey::VariableTensorId:
       return "VariableTensorId";
+    case DispatchKey::BackendSelect:
```

Comment:
Why are we calling this `BackendSelect` and not `CompoundOp`? This should be the same mechanism, right?

Reply:
nvm, saw your comment in the doc